src/app/components/plugin-manager-component.js

```diff
@@ -87,43 +87,40 @@ class PluginManagerComponent extends ViewPlugin {
     this.appManager.event.on('added', () => { this.reRender() })
   }
 
-  renderItem (name) {
-    const api = this.appManager.getOne(name)
-    if (!api) return
-    const isActive = this.appManager.isActive(name)
-    const displayName = (api.profile.displayName) ? api.profile.displayName : name
+  renderItem (profile, isActive) {
+    const displayName = (profile.displayName) ? profile.displayName : profile.name
 
     // Check version of the plugin
     let versionWarning
     // Alpha
-    if (api.profile.version && api.profile.version.match(/\b(\w*alpha\w*)\b/g)) {
+    if (profile.version && profile.version.match(/\b(\w*alpha\w*)\b/g)) {
       versionWarning = yo`<small title="Version Alpha" class="${css.versionWarning} plugin-version">alpha</small>`
     }
     // Beta
-    if (api.profile.version && api.profile.version.match(/\b(\w*beta\w*)\b/g)) {
+    if (profile.version && profile.version.match(/\b(\w*beta\w*)\b/g)) {
       versionWarning = yo`<small title="Version Beta" class="${css.versionWarning} plugin-version">beta</small>`
     }
 
     const activationButton = isActive
       ? yo`
-      <button onclick="${_ => this.appManager.deactivateOne(name)}" class="btn btn-secondary btn-sm">
+      <button onclick="${_ => this.appManager.deactivatePlugin(profile.name)}" class="btn btn-secondary btn-sm">
```

Comment:
If you don't want to repeat `profile.name` & `profile.description` everywhere eyou can do : 
`const { name, description } = profile` at the beginning of the method.